pom.xml

```diff
@@ -183,6 +183,11 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <version>1.54</version>
         </dependency>
+        <dependency>
```

Comment:
please put it in "provided" scope. I like to have Lombok for development, but not in the final jar.